docker/msdp/build_all.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo "Build all MS DeepSpeed flavoured PyTorch-Ignite images"
+
+# Start script from ignite docker folder
+if [ ! -d msdp ]; then
+    echo "Can not find 'msdp' folder"
+    echo "Usage: sh msdp/build_all.sh"
+    exit 1
+fi
+
+curr_dir=$PWD
+cd $curr_dir/msdp
+
+set -xeu
+
+image_name="msdp-apex-base"
+
+docker build -t pytorchignite/${image_name}:latest -f Dockerfile.${image_name} .
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+docker tag pytorchignite/${image_name}:latest pytorchignite/${image_name}:${image_tag}
```

Comment:
Maybe put `image_tag` in a for loop like for `main` and `hvd` scripts.

Reply:
Yes, it was the first script I wrote. As there are 3 cases I though loop will not refactor a lot... 
OK, I can change that.